src/telliot_feed_examples/cli.py

```diff
@@ -265,13 +282,12 @@ async def report(
 
         _ = input("Press [ENTER] to confirm settings.")
 
-        chosen_feed = LEGACY_DATAFEEDS[legacy_id]
-
         common_reporter_kwargs = {
             "endpoint": core.endpoint,
             "private_key": core.get_staker().private_key,
             "master": core.tellorx.master,
             "oracle": core.tellorx.oracle,
+            "sync_feed": sync_feed,
```

Comment:
Debating the separate flag. Probably conflicts with `datafeed`, which is still a required argument to the reporters. 
 We should probably make `datafeed` an optional input.  If None, the default behavior would be to automatically choose the suggested feed.

Reply:
Good idea